### ISSUE TITLE
Firefox Version

### ISSUE THREAD

**Issue by APLA01** [ghostwords/chameleon#25]
hello, i love this addon and i can't find a addon that is on the level of anti-fingerprinting this is

my addons
Noscript
cookie monster
Disconnect
Ublock Origin
Umatrix
Canvas Blocker

i would love to add Chameleon to that list of security addons, this combined with my vpn would make my privacy a whole lot better...

**Comment by ghostwords** [comment]
Hello! Chameleon is a research project, not yet ready for daily use. If you use it in Chrome to make yourself more private, I suggest uninstalling it for now.

Regarding your add-ons: NoScript and uMatrix seem redundant to me. I would keep uMatrix and remove NoScript, if you are comfortable using uMatrix's interface. Cookie Monster's functionality may also be covered by uMatrix.

Likewise, Disconnect and uBlock seem redundant to me ... I would keep uBlock, but that's a personal preference.

We're tracking Chameleon for Firefox in issue #22.

**Comment by APLA01** [comment]
> Regarding your add-ons: NoScript and uMatrix seem redundant to me. I would keep uMatrix and remove NoScript, if you are comfortable using uMatrix's interface. Cookie Monster's functionality may also be covered by uMatrix.

this would seem like a good choice, but uMatrix may have more functionality (i like it more than Noscript) it doesn't support some blocking and forbidding like blocking XSS and using ABE, i also use Noscript to block Java, (till i get a good anti-fingerprinting addon) WebGL, and XSS, i also use ABE and clearclick protection, i also don't have flash player on firefox (i do have it installed, just not on the browser) 

> Likewise, Disconnect and uBlock seem redundant to me ... I would keep uBlock, but that's a personal preference.

while uBlock has Disconnect's lists (except the ad one, though i added it :O) and more, (I have every list enabled [including the language packs] along with extra lists
http://rickrolldb.com/ricklist.txt
https://raw.githubusercontent.com/r4vi/block-the-eu-cookie-shit-list/master/filterlist.txt
https://easylist-downloads.adblockplus.org/fb_annoyances_full.txt
https://s3.amazonaws.com/lists.disconnect.me/simple_ad.txt
http://www.kiboke-studio.hr/i-dont-care-about-cookies/abp/) Disconnect still has a bit more power in blocking with it's blocklists, for example, in "http://www.isitdownrightnow.com/reddit.com.html" uMatrix blocks about 100 stuff, uBlock blocks 4 things, and Disconnect blocks 103 things...

**Comment by APLA01** [comment]
oh and also covering cookie monster, it gets rid of the cookie monster, along with about:config stuff to get rid of supercookies and flash won't do anything :)

**Comment by APLA01** [comment]
nevermind, i have decided to unblock javascript, man i need a good anti-fingerprint addon!!!
